### ISSUE TITLE
Project: SendMail Contact Form using MS Graph API App Only

### ISSUE THREAD

**Issue by zawhtut** [microsoft/hack-together#162]
### Project name

Mail Contact Form

### Description

I have created a Blazor web application with a contact form that allows website visitors to send a message to the website owner. To send the message, I used Microsoft Graph API and app-only authentication so that users do not need to log in. The message is sent directly to the website owner's Outlook email address. This ensures that the website owner is notified immediately of any messages sent through the contact form.

### Repo URL

https://github.com/zawhtut/MailContactForm

### Team members

Zaw Htut

**Comment by waldekmastykarz** [comment]
Thank you for your submission @zawhtut! Would you be willing to record a short video to demonstrate the app? It's not required for the hackathon but it would help us showcase your work internally at Microsoft and publicly in the community 😊

**Comment by zawhtut** [comment]
Yes... Thanks @waldekmastykarz...  I will be doing it now...😃🙏...  This is the minimum viable product for now. So I hard-coded my userId and email in `appsettings`. I was trying to do the next step, which is to register the owner's email with delegated flow for the initial setup time and deregister it later with appropriate mechanisms. My long-term goal is to turn it into a Blazor component that can be packaged in NuGet.

**Comment by zawhtut** [comment]
https://user-images.githubusercontent.com/10856644/225246135-f17529de-43be-46a7-961a-f2661616eb75.mp4
